guides/source/testing.md

```diff
@@ -1255,7 +1264,7 @@ end
 Testing Time-Dependent Code
 ---------------------------
 
-Rails provides inbuilt helper methods that enable you to assert that your time-sensitve code works as expected.
+Rails provides inbuilt helper methods that enable you to assert that your time-sensitive code works as expected.
```

Comment:
http://ell.stackexchange.com/questions/16886/built-in-in-built-inbuilt-and-builtin-im-confused
http://www.merriam-webster.com/interstitial-ad?next=/dictionary/built-in

Going to leave as is, as they are mostly interchangeable and the current one is more suitable in context of programs.